generators/app/templates/src/app/screens/Registration/layout.tsx

```diff
@@ -0,0 +1,94 @@
+  return (
+    <form className={`column center full-width m-top-8 ${styles.formContainer}`} onSubmit={onSubmit}>
+      <div className="column center m-bottom-3">
+        <h1 className="m-bottom-1">{i18next.t('Registration:registration')}</h1>
+        <h2>{i18next.t('Registration:registrationExplanation')}</h2>
+      </div>
+      <div className={`row space-between m-bottom-2 full-width ${styles.sectionContainer}`}>
+        <InputLabel
+          className="full-width"
+          label={i18next.t('Registration:name')}
+          name={FIELDS.name}
+          inputId={FIELDS.name}
+          dataFor={FIELDS.name}
+          inputType="text"
+          inputClassName={`m-right-1 m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:namePlaceholder') as string}
+          handleChange={onInputChange}
+        />
+        <InputLabel
+          className="full-width"
+          label={i18next.t('Registration:surname')}
+          name={FIELDS.surname}
+          inputId={FIELDS.surname}
+          dataFor={FIELDS.surname}
+          inputType="text"
+          inputClassName={`m-left-1 m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:surnamePlaceholder') as string}
+          handleChange={onInputChange}
+        />
+      </div>
+      <div className={`column m-bottom-2 ${styles.sectionContainer}`}>
+        <InputLabel
+          className="full-width"
+          label={i18next.t('Registration:email')}
+          name={FIELDS.email}
+          inputId={FIELDS.email}
+          dataFor={FIELDS.email}
+          inputType="text"
+          inputClassName={`m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:emailPlaceholder') as string}
+          handleChange={onInputChange}
+        />
+      </div>
+      <div className={`row space-between m-bottom-2 full-width ${styles.sectionContainer}`}>
+        <InputLabel
+          className="full-width"
+          label={i18next.t('Registration:password')}
+          name={FIELDS.password}
+          inputId={FIELDS.password}
+          dataFor={FIELDS.password}
+          inputType="text"
+          inputClassName={`m-right-1 m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:passwordPlaceholder') as string}
+          handleChange={onInputChange}
+        />
+        <InputLabel
+          className="full-width"
+          label={i18next.t('Registration:confirmPassword')}
+          name={FIELDS.confirmPassword}
+          inputId={FIELDS.confirmPassword}
+          dataFor={FIELDS.confirmPassword}
+          inputType="text"
+          inputClassName={`m-left-1 m-bottom-2 full-width ${styles.input}`}
+          placeholder={i18next.t('Registration:confirmPasswordPlaceholder') as string}
+          handleChange={onInputChange}
+        />
+      </div>
+      <div className={`column center ${styles.sectionContainer}`}>
```

Comment:
This div is identical to RecoverPassword layout. Maybe you could extract it. It is a suggestion, does it look good? 🤔

Reply:
I thought about this, but since the login and registration almost never look the same it seemed pointless. If anyone else considers that this is necessary I'll gladly do it!